octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -326,9 +332,10 @@ def _get_available_evaluators(self, matrix_id, exchange_name, tentacle_type, use
                 if matrix_id not in self.available_evaluators_cache:
                     self.available_evaluators_cache[matrix_id] = {}
                 if exchange_name not in self.available_evaluators_cache[matrix_id]:
-                    self.available_evaluators_cache[matrix_id][exchange_name] = {
```

Comment:
It was a bug ?

Reply:
yep, cache was not keeping multiple values, there is another issue that I'm fixing at the moment